docs/src/pages/WebRatingPage/Rating.scss

```diff
@@ -0,0 +1,28 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-docs-rating {
+  display: flex;
+  flex-wrap: wrap;
+
+  &-margin {
```

Comment:
Should have 2 `-`s
Also can we name it something like `rating` rather than using a name that describes the effect?

Reply:
Sure thing! My only concern in calling it rating was due to the rating component itself having the same name so didn't want to mess the naming.

Inspiration was taken from `DialogExample.scss` 😄

Reply:
At the end of the day this isn't shipped code, so no one really cares what it's called, but I would like 2 `-`s 😝

Reply:
Done! 😄